### DISCUSSION TITLE
Mirror conformation

### DISCUSSION THREAD

**Discussion by Chungwen36** [rdkit/rdkit#5294] [General]
Does RDkit provide a tool for inverting a molecule to its mirror image (e.g. right-handed to left-handed)? Thanks.

**Comment by wopozka** [comment]
Just copy molecule, iterate over all atoms, pick chiral ones and invert their configuration.

**Reply by Chungwen36** [reply]
any code for sharing? if so, highly appreciated

**Comment by wopozka** [comment]
Well, lets create the simplest code possible:
```
from rdkit import Chem
chiral_mol = Chem.MolFromSmiles('C[C@@H](Cl)CC[C@@H](C)I')
invert_mol = Chem.MolFromSmiles('C[C@@H](Cl)CC[C@@H](C)I')
for atom in invert_mol.GetAtoms():
    atom.InvertChirality()
print(Chem.MolToSmiles(chiral_mol, invert_mol))
```
`C[C@H](Cl)CC[C@H](C)I C[C@@H](Cl)CC[C@@H](C)I`
